mac/Keyman4MacIM/Keyman4MacIM/KMAboutWindow/KMAboutWindowController.m

```diff
@@ -51,7 +51,11 @@ - (IBAction)configAction:(id)sender {
     // High Sierra (10.13.1 - 10.13.3). See: https://bugreport.apple.com/web/?problemID=35422518
     // TODO: This bug should be fixed in 10.13.4, so after that has been available for a few weeks, we
```

Comment:
Can you update the comment as well?

Reply:
updated (TODO removed, next comment updated)